gitlab/mixins.py

```diff
@@ -660,10 +660,11 @@ class ObjectRotateMixin(_RestObjectBase):
         optional=("expires_at",),
     )
     @exc.on_http_error(exc.GitlabRotateError)
-    def rotate(self, **kwargs: Any) -> dict[str, Any]:
+    def rotate(self, self_rotate: bool = False, **kwargs: Any) -> dict[str, Any]:
```

Comment:
Can this be changed to require it to be a keyword-arg?

`def rotate(self, *, self_rotate: bool = False, **kwargs: Any) ->`

Thanks.